src/vm/eventpipebuffermanager.h

```diff
@@ -141,8 +141,8 @@ class EventPipeBufferList
     // If it is false, then this buffer can be de-allocated after it is drained.
     Volatile<bool> m_ownedByThread;
 
-    // True if a thread is writing something to this list
-    Volatile<bool> m_threadEventWriteInProgress;
+    // Lock to protect access to the active buffer
```

Comment:
We might want to be more specific about what "active buffer" is referring to for the unfortunate soul that needs to touch this part of the runtime after us.